### ISSUE TITLE
suggestion: allow nested custom commands

### ISSUE THREAD

**Issue by kaynn** [ForkIssues/TrackerWin#1454]
I'd suggest creating folders based on / in custom command names
ex:
```
[    
    {
        "name": "Open/Cmd1",
        "target": "repository",
    },
    {
        "name": "Open/Cmd2",
        "target": "repository",
    }
]
```
would create a Open folder, with 2 entries: Cmd1 and Cmd2

**Comment by DanPristupov** [comment]
Thank you for the great idea!
We are going to introduce that in 1.83

**Comment by DanPristupov** [comment]
We just released Fork 1.83
